components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/SalesforceEndpointConfig.java

```diff
@@ -143,6 +147,12 @@
     @UriParam
     private String instanceId;
 
+    // Streaming API properties
+    @UriParam
+    private Integer defaultReplayId;
+    @UriParam
+    private Map<String, Integer> initialReplayIdMap;
```

Comment:
As its just a option for setting simple text=number, then we should make it a multi valued type, so you can configure this directly from the uris as well. See for example MailEndpoint for an example. You can then try to do this and do another PR with this improvement.